plugins/TrackerElectronsFeatures.cc

```diff
@@ -210,6 +211,18 @@ TrackerElectronsFeatures::TrackerElectronsFeatures(const ParameterSet& cfg):
 //
 // member functions
 //
+vector<float> 
+TrackerElectronsFeatures::get_iso_rings(const GsfElectronRef& ele, const vector<reco::TrackRef> &tracks) {
+	vector<float> ret = {0., 0., 0., 0.};
+	// for(const auto& trk : tracks) {
```

Comment:
@mverzett, does this include the contribution from the electron itself? i.e., the ktf track from the electron is found in the ```tracks``` collection?

Reply:
yes, it's easier to remove it afterwards than at this stage

Reply:
actually I can try to remove it at this stage already